app/models/photo.rb

```diff
@@ -11,24 +13,19 @@ def shortened_description
   end
 
   def file_location(type = :preview)
-    # byebug if self.store.id == 101
     file_url ? file_url : seed_url(type)
   end
 
-
   def self.active
     where(active: true)
   end
 
-  def self.cat_filter(id)
-    id == "0" ? all : where(category_id: id)
-  end
-
-  def category
-    Category.find(category_id) rescue Category.new(name: "")
-  end
-
-  def filter(id)
+  def self.category_photos(cat_id)
```

Comment:
I think this isn't necessary. Since you know the category id, you can just call Category.find(#).photos.active to get this collection rather than making a whole new method for it.